Makefile

```diff
@@ -39,5 +39,8 @@ check-failures: check-test-lib
 check-squash:
 	./tests/squash/squash.sh
 
-check: check-failures check-squash
+check-latest-imagestream:
+	cp check_imagestreams.py tests/ && cd tests && ./check_imagestreams.sh
```

Comment:
Why the script is copied? Wouldn't it be enough to `cd tests && ../check_imagestreams.sh`?

Reply:
I will try it. For some reason it failed when I have implemented.

Reply:
Good catch. works.